sql/core/src/main/scala/org/apache/spark/sql/functions.scala

```diff
@@ -3282,6 +3282,14 @@ object functions {
    */
   def size(e: Column): Column = withExpr { Size(e.expr) }
 
+  /**
+   * Returns length of array or map as long.
+   *
+   * @group collection_funcs
+   * @since 2.4.0
+   */
+  def cardinality(e: Column): Column = withExpr { Cardinality(e.expr) }
```

Comment:
Do not add the function APIs. Adding the SQL function is enough.

Reply:
I see.